scripts/airgapped/deploy-1.9.sh

```diff
@@ -0,0 +1,182 @@
+mt-broker-filter/filter: $(img knative-releases/knative.dev/eventing/cmd/broker/filter)
+mt-broker-ingress/ingress: $(img knative-releases/knative.dev/eventing/cmd/broker/ingress)
+pingsource-mt-adapter/dispatcher: $(img knative-releases/knative.dev/eventing/cmd/mtping)
+"
+juju deploy --trust --debug ./$(charm knative-operator) --resource knative-operator-image=$(img knative-releases/knative.dev/operator/cmd/operator) --resource knative-operator-webhook-image=$(img knative-releases/knative.dev/operator/cmd/webhook) --config otel-collector-image=$(img otel/opentelemetry-collector)
+juju deploy --trust --debug ./$(charm knative-serving) --config namespace=knative-serving --config istio.gateway.namespace=kubeflow --config istio.gateway.name=kubeflow-gateway \
+    --config queue_sidecar_image="$(img knative-releases/knative.dev/serving/cmd/queue)" \
+    --config custom_images="activator: $(img knative-releases/knative.dev/serving/cmd/activator)
+autoscaler: $(img knative-releases/knative.dev/serving/cmd/autoscaler)
+controller: $(img knative-releases/knative.dev/serving/cmd/controller)
+webhook: $(img knative-releases/knative.dev/serving/cmd/webhook)
+autoscaler-hpa: $(img knative-releases/knative.dev/serving/cmd/autoscaler-hpa)
+net-istio-controller/controller: $(img knative-releases/knative.dev/net-istio/cmd/controller)
+net-istio-webhook/webhook: $(img knative-releases/knative.dev/net-istio/cmd/webhook)
+queue-proxy: $(img knative-releases/knative.dev/serving/cmd/queue)
+domain-mapping: $(img knative-releases/knative.dev/serving/cmd/domain-mapping:)
+domainmapping-webhook: $(img knative-releases/knative.dev/serving/cmd/domain-mapping-webhook)
+migrate: $(img knative-releases/knative.dev/pkg/apiextensions/storageversion/cmd/migrate)
+"
+
+juju deploy --trust --debug ./$(charm kserve-controller) --resource kserve-controller-image=$(img kserve-controller) --resource kube-rbac-proxy-image=$(img kubebuilder/kube-rbac-proxy) --config custom_images="configmap__agent: '$(img kserve/agent)'
+configmap__batcher: '$(img kserve/agent)'
+configmap__explainers__art: '$(img kserve/art-explainer)'
+configmap__logger: '$(img kserve/agent)'
+configmap__router: '$(img kserve/router)'
+configmap__storageInitializer: '$(img kserve/storage-initializer)'
+serving_runtimes__huggingfaceserver: '$(img kserve/huggingfaceserver)'
+serving_runtimes__lgbserver: '$(img kserve/lgbserver)'
+serving_runtimes__kserve_mlserver: '$(img seldonio/mlserver)'
+serving_runtimes__paddleserver: '$(img kserve/paddleserver)'
+serving_runtimes__pmmlserver: '$(img kserve/pmmlserver)'
+serving_runtimes__sklearnserver: '$(img kserve/sklearnserver)'
+serving_runtimes__tensorflow_serving: '$(img tensorflow/serving)'
+serving_runtimes__torchserve: '$(img pytorch/torchserve-kfs)'
+serving_runtimes__tritonserver: '$(img nvcr.io/nvidia/tritonserver)'
+serving_runtimes__xgbserver: '$(img kserve/xgbserver)'
+"
```

Comment:
during testing, the deployment command I got was:
```
13:26:56 DEBUG juju.cmd supercommand.go:57   args: []string{"/snap/juju/28056/bin/juju", "deploy", "--trust", "--debug", "./kserve-controller_eb87729.charm", "--resource", "kserve-controller-image=172.17.0.2:5000/charmedkubeflow/kserve-controller:0.13.0-e151c86", "--resource", "kube-rbac-proxy-image=172.17.0.2:5000/kubebuilder/kube-rbac-proxy:v0.13.1", "--config", "custom_images=configmap__agent: ''\nconfigmap__batcher: ''\nconfigmap__explainers__art: '172.17.0.2:5000/kserve/art-explainer:latest'\nconfigmap__logger: ''\nconfigmap__router: '172.17.0.2:5000/kserve/router:v0.13.0'\nconfigmap__storageInitializer: ''\nserving_runtimes__huggingfaceserver: '172.17.0.2:5000/kserve/huggingfaceserver:v0.13.0'\nserving_runtimes__lgbserver: ''\nserving_runtimes__kserve_mlserver: '172.17.0.2:5000/seldonio/mlserver:1.3.2'\nserving_runtimes__paddleserver: ''\nserving_runtimes__pmmlserver: ''\nserving_runtimes__sklearnserver: ''\nserving_runtimes__tensorflow_serving: '172.17.0.2:5000/tensorflow/serving:2.6.2'\nserving_runtimes__torchserve: '172.17.0.2:5000/pytorch/torchserve-kfs:0.9.0'\nserving_runtimes__tritonserver: ''\nserving_runtimes__xgbserver: ''\n"}
```
the following images are missing from `custom_images`:
* configmap__agent
* configmap__batcher
* configmap__logger
* configmap__storageInitializer
* serving_runtimes__lgbserver
* serving_runtimes__paddleserver
* serving_runtimes__pmmlserver
* serving_runtimes__sklearnserver
* serving_runtimes__tritonserver
* serving_runtimes__xgbserver

Reply:
This is because of the rock integration we did for `kserve`, I updated the deployment script to reflect that.